repos/system_upgrade/common/actors/filterrpmtransactionevents/actor.py

```diff
@@ -29,11 +29,15 @@ def process(self):
         to_remove = set()
         to_keep = set()
         to_upgrade = set()
+        modules_to_enable = set()
+        modules_to_disable = set()
         for event in self.consume(RpmTransactionTasks, PESRpmTransactionTasks):
             local_rpms.update(event.local_rpms)
             to_install.update(event.to_install)
             to_remove.update(installed_pkgs.intersection(event.to_remove))
             to_keep.update(installed_pkgs.intersection(event.to_keep))
+            modules_to_enable.update(event.modules_to_enable)
+            modules_to_disable.update(event.modules_to_enable)
```

Comment:
This might be a typo:
```suggestion
            modules_to_disable.update(event.modules_to_disable)
```